vllm/lora/layers.py

```diff
@@ -1067,6 +1067,10 @@ def org_vocab_size(self):
     def include_gpu_probs_tensor(self):
         return self.base_layer.include_gpu_probs_tensor
 
+    @property
+    def should_modify_greedy_probs_inplace(self):
```

Comment:
Confused why this is here, why does this layer need it?

Reply:
I'm not sure, I just greped for all the places that `include_gpu_probs_tensor` was used and added the change there to keep semantics identical. See the lines above.